src/main/java/vectorwing/farmersdelight/common/world/VillageStructures.java

```diff
@@ -3,9 +3,9 @@
 import com.mojang.datafixers.util.Pair;
 import net.minecraft.core.Registry;
 import net.minecraft.resources.ResourceLocation;
-import net.minecraft.world.level.levelgen.feature.structures.SinglePoolElement;
-import net.minecraft.world.level.levelgen.feature.structures.StructurePoolElement;
-import net.minecraft.world.level.levelgen.feature.structures.StructureTemplatePool;
+import net.minecraft.world.level.levelgen.structure.pools.SinglePoolElement;
```

Comment:
I do suspect other changes may be needed here, but my experience is lacking with StructureGen

Reply:
Yeah, some changes will be needed in this file, it's erroring on structure gen. But I got it working on my repo, so I'll do it later. 👍